deploy/helm/internal-charts/storage-api/Chart.lock

```diff
@@ -1,6 +0,0 @@
-dependencies:
```

Comment:
We shouldn't delete the lock file. And I think your PR doesn't need to touch `internal-charts`